internal/handler/beer.go

```
@@ -165,11 +165,14 @@ func (h WorkspaceServer) ListBeers(reqResp *web.ReqRespPair) error {
 		Page:  page,
 		Limit: 20, //nolint:mnd
 	}
-	pagination, paginationErr := h.beerService.PaginateBeers(filter)
+	pagination, paginationErr := h.beerService.PaginateBeers(reqResp.Request.Context(), filter)
 	if paginationErr != nil {
 		return apperr.NewInternalServerError("Failed to paginate beers", paginationErr)
 	}
 
+	fmt.Println(pagination.Page)
```

golangci-lint (2.2.2)

use of `fmt.Println` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$`

(forbidigo)